object_relations/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: object_relations
+description: A simple command-line application.
+version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.16.0-134.1.beta <3.0.0'
```

Comment:
I think you should loosen this constraint to >= 2.15.0, so the example will work with the current stable